### PR TITLE
New buttons and colors

### PR DESCRIPTION
**Pull request by AdnanDervisevic**

### CODE REVIEW

File: src/common/components/Button/README.md [hunk 1]
```diff
@@ -4,15 +4,15 @@ Buttons are used to invoke an event.
...
-| `disabled`      |          | boolean       |                                                                                     |                                                                  |
-| `loading`       |          | boolean       |                                                                                     |                                                                  |
+| Name            | Required | Type          | Possible values                                                                                 | Notes                                                            |
+| --------------- | -------- | ------------- | ----------------------------------------------------------------------------------------------- | ---------------------------------------------------------------- |
+| `kind`          |          | string        | `'primary'`, `'purple'`, `'secondary'`, `'secondaryNegative'`, `'success'`, `'warn'`, `'text'`, | Use `secondaryNegative` only if you have an off white background |
```
Comment (svenheden) [inline src/common/components/Button/README.md]: this change doesn't correspond with the `ButtonKind` type below

Comment (AdnanDervisevic) [reply src/common/components/Button/README.md]: Yep, noticed it myself actually and commited a fix :p

Comment (svenheden) [reply src/common/components/Button/README.md]: 👍🏼😀

**Pull request merged**